fhirstarter/tests/config.py

```diff
@@ -99,21 +129,29 @@ def app(provider: FHIRProvider) -> TestClient:
     return TestClient(app)
 
 
-def client() -> TestClient:
-    """Create an app that provides all FHIR interactions."""
+def create_test_client(
```

Comment:
This function makes it easy to create an app with any permutation of interactions, and in either async or nonasync mode.